api/rpc/logs/logs.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package logs;
```

Comment:
I use the package 'logs' (plural) because using 'log' (singludar) conflicts with the builtin go 'log' package.